service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FileUploadNotificationReceiver.java

```diff
@@ -28,7 +28,7 @@
      * @param sasToken The iot hub SAS token for the given device
      * @param iotHubServiceClientProtocol The iot hub protocol name
      */
-    FileUploadNotificationReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol)
+    FileUploadNotificationReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol, ProxyOptions proxyOptions)
     {
```

Comment:
Isn't this a breaking change?

Reply:
As per above, I'll add an overload to take the proxy instead